src/widgets/menu/SideMenuItem.m.css

```diff
@@ -1,14 +1,92 @@
 @import '../../variables.css';
 
 .root {
+	position: relative;
+	text-transform: none;
+}
+
+.link {
 	font-size: 0.95rem;
 	line-height: 1.65;
 	color: #999;
 	padding: calc(var(--grid-size) / 2) 0;
-	display: block;
+	display: inline-block;
 	text-decoration: none;
+	position: relative;
 }
 
-.root.selected {
+.link:hover {
 	color: #333;
 }
+
+.link:hover,
+.link:focus {
+	text-decoration: none;
+}
+
+.link::before {
+	content: '';
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 100%;
+	border-bottom: 1px solid var(--color-dark-gray);
+	transition: right 0.3s ease-out;
+}
+
+.link:hover::before {
+	right: 0;
+}
+
+@media (max-width: 768px) {
+	.link::before {
+		content: none;
+	}
+
+	.link:hover {
+		color: inherit;
+	}
+}
+
+.dropdownLink.selected,
+.link.selected {
+	color: #333;
+}
+
+.dropdownLink.selectedInverse,
+.link.selectedInverse {
+	color: #fff;
```

Comment:
Setting a foreground color to white without also setting the background color has the potential to result in unreadable text.

Reply:
Good point